targetd.te

```diff
@@ -25,13 +25,14 @@ files_tmp_file(targetd_tmp_t)
 
 
 
-allow targetd_t self:capability { ipc_lock sys_admin sys_nice };
+allow targetd_t self:capability { chown ipc_lock sys_admin sys_nice };
```

Comment:
Do you know for which files this capability is required?

Reply:
See https://github.com/fedora-selinux/selinux-policy-contrib/pull/334#issuecomment-694997933